src/zkevm_specs/evm/typing.py

```diff
@@ -817,23 +819,25 @@ def _append_steps(
                 exponent = exponent - 1
 
     def _append_padding_row(self, identifier: IntOrFQ):
-        self.rows.append(
-            ExpCircuitRow(
-                q_usable=FQ.zero(),
-                is_step=FQ.zero(),
-                identifier=FQ(identifier),
-                is_last=FQ.zero(),
-                base=RLC(0),
-                exponent=RLC(0),
-                exponentiation=RLC(0),
-                a=RLC(0),
-                b=RLC(0),
-                c=RLC(0),
-                d=RLC(0),
-                q=RLC(0),
-                r=RLC(0),
+        rows_left = self.MAX_EXP_STEPS - len(self.rows)
+        for i in range(rows_left):
+            self.rows.append(
+                ExpCircuitRow(
+                    q_usable=FQ.one(),
+                    is_step=FQ.zero(),
+                    identifier=FQ(identifier),
```

Comment:
Once you move this out to the `fill_dummy_events` as mentioned above, the `identifier` can be `0` in this case. The identifier is basically the RW counter at which the exponentiation event starts (and is used to do lookups from the EVM circuit). But for the dummy events of `1 ^ 1 == 1` we wouldn't need an identifier as they are dummy events that won't be looked up to.